### ISSUE TITLE
[Suggestion] Hide buffers tilde by default

### ISSUE THREAD

**Issue by polirritmico** [shortcuts/no-neck-pain.nvim#238]
Hi, great plugin! Just want to point a little suggestion.

When a transparent background is used the lateral buffers show the tilde on the empty lines (at least that was the case with my config). I think that the expected default is without those tildes on the borders so the `bo.fillchars` could be setted to `"eob:  "` by default. Currently this setting is needed:

```lua
require("no-neck-pain").setup({
    buffers = {
        wo = {
            fillchars = "eob: "
        },
    },
})
```

If not, maybe point it out in the wiki showcase?

---

## Current:
![imagen](https://github.com/shortcuts/no-neck-pain.nvim/assets/24460484/690722cd-d9be-4acd-8248-87ef92e5ff82)

## Expected:
![imagen](https://github.com/shortcuts/no-neck-pain.nvim/assets/24460484/784ea022-065b-4fc8-835c-4487601db8a2)

Reggards

**Comment by shortcuts** [comment]
Hi @polirritmico, glad you like the plugin and thanks for the feedback!

> I think that the expected default is without those tildes on the borders so the bo.fillchars could be setted to "eob:  " by default.

I tried to make side buffers behave similarly to the main buffer, so that the plugin doesn't override theme settings without the user's agreement. Looking at your first screenshot, you theme seems to display theme so I believe it behaves correctly

> If not, maybe point it out in the wiki showcase?

Definitely! Added a small explanation here: https://github.com/shortcuts/no-neck-pain.nvim/wiki/Showcase#hide-end-of-buffers-eob-characters-in-side-buffers

I've used your screenshot, let me know if you don't want to, I'll create a new one
